auth/store_test.go

```diff
@@ -230,6 +231,126 @@ func TestRoleAdd(t *testing.T) {
 	}
 }
 
+func setupAuthStore(t *testing.T) (AuthStore, func(t *testing.T)) {
```

Comment:
if we want to have setup, we should user it everywhere in this pkg. we need to make other tests consistent. probably you can submit a pr to make this happen first.